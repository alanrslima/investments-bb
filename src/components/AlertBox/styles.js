import { StyleSheet } from 'react-native';
import { fonts, colors } from '../../styles';

export default StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    padding: 50,
  },
  alertText: {
    textAlign: 'center',
    fontSize: fonts.DESCRIPTION,
    color: colors.GRAY_DARK,
  }
});
