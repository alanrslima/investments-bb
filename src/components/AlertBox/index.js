import React from 'react';
import { Text } from 'react-native';
import * as Animatable from 'react-native-animatable';

import styles from './styles';

export default function AlertBox({ message }) {
  return (
    <Animatable.View
      animation="fadeInUp"
      iterationCount={1}
      style={styles.container}
    >
      <Text style={styles.alertText}>{message}</Text>
    </Animatable.View>
  );
}
