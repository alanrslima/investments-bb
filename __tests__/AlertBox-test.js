import React from 'react';
import renderer from 'react-test-renderer';
import AlertBox from '../src/components/AlertBox';


test('renders correctly', () => {
  const tree = renderer.create(<AlertBox message="Message alert" />).toJSON();
  expect(tree).toMatchSnapshot();
});
