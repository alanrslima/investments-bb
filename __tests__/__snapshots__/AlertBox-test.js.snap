// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  forwardedRef={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 0,
      "padding": 50,
      "transform": Array [
        Object {
          "translateY": 100,
        },
      ],
    }
  }
>
  <Text
    style={
      Object {
        "color": "#707070",
        "fontSize": 15,
        "textAlign": "center",
      }
    }
  >
    Message alert
  </Text>
</View>
`;
